x/pairing/types/message_stake_provider.go

```diff
@@ -49,13 +51,13 @@ func (msg *MsgStakeProvider) ValidateBasic() error {
 		return sdkerrors.Wrapf(legacyerrors.ErrInvalidAddress, "invalid creator address (%s)", err)
 	}
 
-	if msg.Moniker == "" {
-		return sdkerrors.Wrapf(MonikerEmptyError, "invalid moniker (%s)", msg.Moniker)
-	}
-
 	if len(msg.Moniker) > MAX_LEN_MONIKER {
 		return sdkerrors.Wrapf(MonikerTooLongError, "invalid moniker (%s)", msg.Moniker)
 	}
 
+	if msg.DelegateCommission > 100 {
+		return sdkerrors.Wrapf(DelegatecommissionOOBError, "commission out of bound (%d)", msg.DelegateCommission)
+	}
+
 	return nil
```

Comment:
also check the `DelegateLimit` is valid (coin).

Reply:
done 0aa5c663504a1f29e262ff8666deee1af71ce302